pokemongo_bot/cell_workers/recycle_items.py

```diff
@@ -99,7 +99,7 @@ def _schedule_next_force(self):
         self._next_force = self._get_next_force_schedule()
         self.emit_event(
             'next_force_recycle',
-            formatted="Next forced item recycle at {time}",
+            formatted="Next forced item recycle at {}".format(str(self._next_force.strftime("%H:%M:%S"))),
```

Comment:
During refactor I was getting an error with force_item_recycle. By changing the formatting the error went away.